tests/suites/0_stateless/03_dml/03_0019_select_empty.result

```diff
@@ -0,0 +1 @@
+0
```

Comment:
How about adding one test statement before `/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;`?
```
/*!40101*/select number from numbers_mt(2);
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101*/select number from numbers_mt(1);
```

and check 
```
0
1
0
```

Maybe we can be more sure where is the empty line.

Reply:
:+1: 
This draft PR only for test, found some test issues, still would like you can push a new PR for it.

Reply:
@BohuTANG  pushed on https://github.com/datafuselabs/databend/pull/4894